server/graphql/vvm/src/types/vvm_status_log.rs

```diff
@@ -0,0 +1,95 @@
+use async_graphql::{dataloader::DataLoader, *};
+use chrono::{DateTime, Utc};
+use graphql_core::{
+    loader::{StockLineByIdLoader, UserLoader, VVMStatusByIdLoader},
+    ContextExt,
+};
+use graphql_types::types::{StockLineNode, UserNode};
+use repository::vvm_status::vvm_status_log_row::VVMStatusLogRow;
+
+use super::vvm_status::VVMStatusNode;
+
+#[derive(PartialEq, Debug)]
+pub struct VVMStatusLogNode {
+    vvm_status_log: VVMStatusLogRow,
+}
+
+#[Object]
+impl VVMStatusLogNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
+    pub async fn status(&self, ctx: &Context<'_>) -> Result<Option<VVMStatusNode>> {
```

Comment:
Should be a vector of status logs please~...

Reply:
Do we want everything or the only ones that are active? Going for active unless we want all